examples/storm-starter/src/jvm/storm/starter/BlobStoreAPIWordCountTopology.java

```diff
@@ -0,0 +1,288 @@
+        return sentence;
+    }
+
+    private static Set<String> getRandomWordSet() {
+        Set<String> randomWordSet = new HashSet<>();
+        Random random = new Random();
+        String[] words = new String[]{ "cow", "jumped", "over", "the", "moon", "apple", "day", "doctor", "away",
+                "four", "seven", "ago", "snow", "white", "seven", "dwarfs", "nature", "two" };
+        // Choosing atmost 5 words to update the blacklist file for filtering
+        for (int i=0; i<5; i++) {
+            randomWordSet.add(words[random.nextInt(words.length)]);
+        }
+        return randomWordSet;
+    }
+
+    private static Set<String> parseFile(String fileName) throws IOException {
+        File file = new File(fileName);
+        Set<String> wordSet = new HashSet<>();
+        if (!file.exists()) {
+            return wordSet;
+        }
+        StringTokenizer tokens = new StringTokenizer(readFile(file).toString(), "\r\n");
+        while (tokens.hasMoreElements()) {
+            wordSet.add(tokens.nextToken());
+        }
+        LOG.info("parseFile {}", wordSet);
+        return wordSet;
+    }
+
+    private static StringBuilder readFile(File file) throws IOException {
+        String line;
+        StringBuilder fileContent = new StringBuilder();
+        // Do not use canonical file name here
+        BufferedReader br = new BufferedReader(new FileReader(file));
+        while ((line = br.readLine()) != null) {
+            fileContent.append(line);
+            fileContent.append(System.lineSeparator());
+        }
+        return fileContent;
+    }
+
+    // Creating a blacklist file to read from the disk
+    public static File createFile(String fileName) throws IOException {
+        File file = null;
+        file = new File(fileName);
+        if (!file.exists()) {
+            file.createNewFile();
+        }
+        writeToFile(file, getRandomWordSet());
+        LOG.info(readFile(file).toString());
+        return file;
+    }
+
+    // Updating a blacklist file periodically with random words
+    public static File updateFile(File file) throws IOException {
+        writeToFile(file, getRandomWordSet());
+        return file;
+    }
+
+    // Writing random words to be blacklisted
+    public static void writeToFile(File file, Set<String> content) throws IOException{
+        FileWriter fw = new FileWriter(file, false);
+        BufferedWriter bw = new BufferedWriter(fw);
+        Iterator<String> iter = content.iterator();
+        while(iter.hasNext()) {
+            bw.write(iter.next());
+            bw.write(System.lineSeparator());
+        }
+        bw.close();
+    }
+
+    public static void main(String[] args) {
+        prepare();
+        BlobStoreAPIWordCountTopology wc = new BlobStoreAPIWordCountTopology();
+        try {
+            File file = createFile(fileName);
+            // Creating blob again before launching topology
+            createBlobWithContent(key, store, file);
```

Comment:
Yes I am catching the keyAlreadyExists exception, I might well have it deleted but the topology will run with no filtering